Indentation checkstyle does not correctly work with java 8

When I wrote a code with java 8 lambda function: 

``` java
    final OperatorChainJob firstJob = () -> {
      final long startTime = System.currentTimeMillis();
      while (System.currentTimeMillis() - startTime < 2000) {
        // empty
      }
      outputs.add(0);
      System.out.println("while-op");
    };
```

The checkstyle complains that 

```
[ERROR] src/test/java/edu/snu/mist/task/executor/ExecutorJobSchedulingTest.java[60] (indentation) Indentation: 'block' child have incorrect indentation level 6, expected level should be 8.
```

But, when I indent the line 60 with indentation level 8, it still complains 

```
[ERROR] src/test/java/edu/snu/mist/task/executor/ExecutorJobSchedulingTest.java[60] (indentation) Indentation: 'block' child have incorrect indentation level 8, expected level should be 6.
```

I think the indentation checkstyle does not correctly work with java 8.
https://github.com/cmssnu/mist/blob/master/src/main/resources/checkstyle.xml#L189
we need to disable it.

--- a/src/main/resources/checkstyle.xml
+++ b/src/main/resources/checkstyle.xml
@@ -185,12 +185,12 @@
 
         <!-- Miscellaneous other checks.                   -->
         <!-- See http://checkstyle.sf.net/config_misc.html -->
+        <!--
         <module name="ArrayTypeStyle"/>
         <module name="Indentation">
             <property name="basicOffset" value="2"/>
             <property name="caseIndent" value="0"/>
         </module>
-        <!--
         <module name="TodoComment">
             <property name="severity" value="warning"/>
             <property name="format" value="TODO[^\[][^J][^I][^R][^A]"/>

